test/models/snapshot_test.rb

```diff
@@ -83,8 +83,12 @@ def test_build_snapshot_item
 
     assert snapshot_item.new_record?
 
+    assert snapshot_item.object['status'] == 0
+
     assert_equal @snapshot.id, snapshot_item.snapshot_id
 
+    assert_equal @snapshot.snapshot_items.first.object['status'], snapshot_item.object['status']
```

Comment:
Can we put these assertions in their own defined test case instead of using the existing one.